### PR TITLE
Add compat for BigInt

### PR DESCRIPTION
**Pull request by VFDan**
The [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) compatibility table is needed: 
- I added a compatibility table for BigInt.
- The CanIUse:  https://caniuse.com/#feat=bigint
Sorry, the file should be in the javascript/builtins/ folder.

### CODE REVIEW

File: BigInt.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+			        ]
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "nodejs": {
+              "version_added": null
```
Comment (chicoxyzzy) [inline BigInt.json]: 10.4.0

**Comment by VFDan** [comment]
I added a commit with the Node.js compatibility, and put it in the correct folder

**Comment by VFDan** [comment]
I un-localized the URL.

File: javascript/builtins/BigInt.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
+  "javascript": {
+    "builtins": {
+      "BigInt": {
+        "__compat": {
+          "mdn_url": " https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/BigInt",
```
Comment (ddbeck) [inline javascript/builtins/BigInt.json]: The leading space before `http` is causing the linter to fail. Removing the space should fix it

File: javascript/builtins/BigInt.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "65",
+			        "flags": [
+			        {
```
Comment (ddbeck) [inline javascript/builtins/BigInt.json]: This block should be indented one level more, like this:

```
"flags": [
  {
    "type": …
```

File: javascript/builtins/BigInt.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+              "version_added": "67"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+    }
```
Comment (ddbeck) [inline javascript/builtins/BigInt.json]: The indentation of these last few braces got a bit jumbled. Could you tidy this up?

**Comment by VFDan** [comment]
I added a commit with those style changes.

**Comment by VFDan** [comment]
Travis CI says: "
```
  File : javascript/builtins/BigInt.json
  Style – Error on line #1
    Actual:   {
    Expected: {
```
"
But when I look at the file, there is nothing to the left of the {. I tried to backspace, but it did nothing.

**Comment by ddbeck** [comment]
It appears that your line endings aren't just new lines (i.e., your line endings are Windows rather than Unix style). I've opened PR #3231 to highlight this better. In the mean time, there's probably an option in your editor to convert line endings, which should fix the problem.

**Comment by VFDan** [comment]
I changed the line endings (I said Unicode but meant Unix).

**Comment by ddbeck** [comment]
@VFDan I went ahead and fixed the line endings directly (and unnoticed tabs vs spaces issue). I don't know why that had to be such a hassle, but hopefully #3231 will avoid that in the future. I'll be merging your contribution shortly. Thank you! 🎉

**Comment by VFDan** [comment]
Thank you for merging the commit! I'm also on MDN under the username VFDan.

**Comment by VFDan** [comment]
I put the compatibility table in the MDN BigInt page.

**Pull request merged**